### PR TITLE
fix: pass on error to callback for better error messages

### PR DESCRIPTION
**Pull request by vinteo**
**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

- [x] Bugfix

- Duplicate of https://github.com/webpack-contrib/karma-webpack/pull/215

**What is the new behavior?**

A proper error is given

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

---
### Issues 

- Fixes https://github.com/webpack-contrib/karma-webpack/issues/214

### PRs

- Duplicate of https://github.com/webpack-contrib/karma-webpack/pull/215

> edited by @michael-ciniawsky (Formatting)

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
@vinteo Please sign the CLA by closing and reopening the PR to trigger the CLA bot again

**Comment by michael-ciniawsky** [comment]
Duplicate of #235

**Comment by vinteo** [comment]
@michael-ciniawsky thanks, let me know if there is anything else I need to do to get this in 😃

**Comment by michael-ciniawsky** [comment]
@vinteo You need to sign the CLA please, close and reopen the PR to trigger the CLA Bot again.

**Comment by vinteo** [comment]
@michael-ciniawsky I have signed it, its the travis that's not triggering or am I missing something?

**Comment by michael-ciniawsky** [comment]
😅  Yeah... my bad all good 👍

**Comment by vinteo** [comment]
@evilebottnawi There are no tests in this library so I wasn't fully sure where to start but I created a very simple test for this very specific scenario

File: test/unit/plugin.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {assert} from 'chai'
```
Comment (evilebottnawi) [inline test/unit/plugin.test.js]: Let's avoid using `unit` directory for test file.

Comment (vinteo) [reply test/unit/plugin.test.js]: It is an existing folder and the current test config doesn't support it in the root `test` folder. What name do you suggest?

Comment (evilebottnawi) [reply test/unit/plugin.test.js]: @vinteo oh sorry, i think we update tests (migrate to `jest`) in this repo, do nothing, leave everything as is :+1:

**Comment by d3viant0ne** [comment]
@vinteo - Can you rebase in latest master when you have a moment, solves the issue with Travis not running PRs

**Comment by vinteo** [comment]
@d3viant0ne done! 😃

**Comment by d3viant0ne** [comment]
Thanks :)

**Pull request merged**